client/src/pages/login.component.js

```diff
@@ -1,13 +1,24 @@
 import React, { Component } from 'react';
 import LoginForm from '../components/login-form.component.js';
+import { userContext } from '../App.js';
 
 export default class Login extends Component {
   render() {
     return (
-      <div className="m-4">
-        <h1 className="text-center">Login</h1>
-        <LoginForm />
-      </div>
+      <>
+        <userContext.Consumer>
+          {({ loginUser }) => {
+            return (
+              <>
+                <div className="m-4">
+                  <h1 className="text-center">Login</h1>
+                  <LoginForm updateUser={loginUser} />
```

Comment:
Not quite sure how this is working would love a tour (between this file and client/src/components/login-form.component.js) You're passing the logged in user id as a property to the login form component?? Or is this 'passing data up' once they log in, the property is set, and ultimately held at the client/src/App.js level??